Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabIntegrationManager.cpp

```diff
@@ -551,7 +631,7 @@ namespace AzToolsFramework
         {
             s_prefabFocusPublicInterface->FocusOnOwningPrefab(AZ::EntityId());
         }
-
+        
```

Comment:
Not sure but this looks like weird spacing here?

Reply:
Fixed!